docs/core-concepts/api-calls-anatomy/index.md

```diff
@@ -3,39 +3,51 @@ title: API Calls Anatomy
 hide_title: true
 draft: false
 sidebar_label: API Calls anatomy
-sidebar_position: 5
+sidebar_position: 1
 tags:
   - concept
 keywords:
   - trading
   - concept
 description: Deriv API calls Anatomy
 ---
+## Subscribe and Send
 
-## API Call Types
+Every API call has the send functionality and several API calls provide the subscribe functionality.
 
-We provide two types of API calls, `Subscription` and `Send Once` ( Fire and Forget )
+### Subscribe
 
-### Subscription
+Several API calls provide the `subscribe` functionality. When you subscribe to an API call, you will receive a continuous stream of data of this particular API call. An example of an API call you can subscribe too, is [Tick History](https://api.deriv.com/api-explorer#ticks_history). If you subscribe to this call, every second or so, you will get a new updated set of tick history data.
 
-Some API calls provide `subscription` ability, which means when you subscribe to them every time that particular event happens for example [Tick History](https://api.deriv.com/api-explorer#ticks_history), these API calls have an `optional` `subscription` field, which if you pass `1` to them, the subscription starts. most of the time data provided by this type of calls will be considered as data source for other calls or features.
+In the message stream from `subscribe` there is a field called `subscription`, this is the `Stream ID`. with this ID you can identify the message stream in your logic and stop the stream with `Forget` and `Forget All` API calls.
 
-### Send Once
+These API calls have an `optional` `subscribe` field, and if you pass `1` to them, the subscription will start and server will continue to send the requested data until you call the `forget`  for your API call you want to `unsubscribe` or `forget`.
```

Comment:
Text change suggestion:
Several API calls provide the `subscribe` functionality. When you subscribe to an API call, you will receive a continuous stream of data of this particular API call.  

Some of these API calls automatically subscribes (e.g. [ticks](https://api.deriv.com/api-explorer#ticks)), and some have an `optional` `subscribe` field, and if you pass `1` to them, the subscription will start and server will continue to send the requested data until you unsubscribe by calling the `forget` or `forget all` API calls.

For example, you can call [Tick History](https://api.deriv.com/api-explorer#ticks_history) to receive tick history data. But when you add the `subscribe` option to this call, you will receive the tick history data you requested in the first response, and you will continue to receive a new response every time there is a new tick published by the server for the given symbol.

In the message stream from `subscribe` there is a field called `subscription`, this is the `Stream ID`. with this ID you can identify the message stream in your logic and stop the stream with `Forget` and `Forget All` API calls.

Reply:
Some API calls will subscribe automatically without passing the subscribe filed.
Different symbols have different publishing frequencies so better not to mention specific frequency.